src/System.Composition.sln

```diff
@@ -0,0 +1,97 @@
+﻿
```

Comment:
All the other folders in \src seem to follow the "folder == nuget package" pattern, so if the assemblies ship in separate packages then it's fine as is with multiple folders (with the sln being in \src\System.Composition as I think that's the most logical place).